.github/workflows/e2e-polybft.yml

```
@@ -6,13 +6,15 @@ on:  # yamllint disable-line rule:truthy
       - main
       - develop
   pull_request:
+  workflow_call:
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       E2E_TESTS: true
       E2E_LOGS: true
+      E2E_TESTS_TYPE: 'E2E Integration'
```

IMO, rename it to `Integration`, it's more consistent.
```suggestion
      E2E_TESTS_TYPE: 'Integration'
```

---

fixed: https://github.com/0xPolygon/polygon-edge/pull/1276/commits/0ddef0e11a39630ccf0ccf5da5eb1a007c211b49